app/src/main/res/values/attr.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="StarView">
+        <attr name="line_color" format="color" />
```

Comment:
Properties of custom views should be prefixed by the view: `starview_line_color`
If I remember correctly, if you try to define the same styled attribute (i.e. `line_color`) in two different views you'll have a conflict (possibly even a compilation error).